README.md

```diff
@@ -4,33 +4,38 @@
 
 Training n°1 for NFT marketplace
 
-Introduction : Business objects managed by a blockchain are called `assets`. On Tezos you will find the term `Financial Asset or FA` with different version 1, 2 or 2.1.
+# Introduction
```

Comment:
I feel like this introduction is not an introduction.
Your real introduction is in the next part:
> We are going to build a wine marketplace

I think it should be `preface` or `foreword` (don't know which one is the good translation of `avant-propos`).